doc/src/content/atoms/button/FixedSize.md

```diff
@@ -6,10 +6,11 @@ title: "Fixed Size Button with color"
 ---
 
 <Box>
-  <Button width="20px" px="100px">
-    20px
+  <Button width="200px" px="50px" mr={'20px'}>
```

Comment:
I felt that if we set width in button, we need to manually adjust padding to make it center aligned. Is this the intended feature ?

Reply:
It is center-aligned because of `textAlign: center` property on button. Padding should not make an issue unless we set inconsistent left and right padding(like: padding-left: 50px, padding-right: 200px)